.github/workflows/publish_osdk_and_ostd.yml

```diff
@@ -48,27 +48,45 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Check Publish OSTD and the test runner
-        # On pull request, set `--dry-run` to check whether OSDK can publish
+      - name: Publish check of OSTD and the test runner
+        # On pull request, set `--dry-run` to check whether they can be published
         if: github.event_name == 'pull_request'
```

Comment:
There are too many duplicated commands between `github.event_name == 'push'` and `github.event_name == 'pull_request'`. Now is the time to wrap them into a utility script under `tools/github_workflows`.

```
Publish OSDK and OSTD to crates.io

Usage: publish_osdk_and_ostd.sh [--target TARGET][--dry-run]
```